src/tests/ftest/util/general_utils.py

```diff
@@ -495,9 +495,13 @@ def stop_processes(hosts, pattern, verbose=True, timeout=60):
             "sudo pkill {}".format(pattern),
             "sleep 5",
             "if pgrep --list-full {}".format(pattern),
+            "then pkill --signal ABRT {}".format(pattern),
+            "sleep 1",
+            "if pgrep --list-full {}".format(pattern),
```

Comment:
The normal agent and server stopping process goes through [`stop()`](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L187).  Here a INT signal is sent and, if needed, it is followed by a KILL signal, as controlled by [this list](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L197).

The `stop_processes()` method is only called when
1. ensuring there are no existing server processes running on the hosts on which the server is about to be started ([here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils.py#L361))
2. an error is detected during server startup ([here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils.py#L487) and [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils.py#L507))
3. as a catch-all for any processes not stopped by `stop()` ([here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils.py#L596) and [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/agent_utils.py#L205))

In typically operation, even though `stop_processes()` is called it is essentially a no-op as the initial `prep` in the command string does not match anything.

I suspect that the `ABRT` signal should instead be added to [`stop()`](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L197) to get the intended result.

Reply:
Hmm. But `stop()` doesn't send SIGTERM to (daos_agent|daos_server), does it? It sends it to the parent orterun process. I don't think that sending SIGABRT to orterun would be passed down its child process(es).

The intent behind adding the SIGABRT signal here was to provide additional information in diagnosing whatever happened in https://jira.hpdd.intel.com/browse/DAOS-6180, which if I understand correctly has already gotten to `stop_processes()`.